### ISSUE TITLE
Return All fields in Autotype

### ISSUE THREAD

**Issue by dploeger** [keeweb/keeweb#1874]
**Is your feature request related to a problem? Please describe.**
I have several entries that use the website field. Instead of keeping additional favorites in my browser, I'd like to use autotype to type the URL from Keeweb.

**Describe the solution you'd like**
The "..." (Shift+Enter) menu should include *all* fields from the entry and not an arbitrary list of certain fields. Currently, the website URL is not part of that.

**Comment by antelle** [comment]
Makes sense for the url field, let's add it to the list as well.

**Comment by dploeger** [comment]
Why not simply use every field?

**Comment by dploeger** [comment]
Maybe then blocklist those entries instead of a allowlisting the other ones?

**Comment by antelle** [comment]
Yes, that's how exactly it's implemented: it's all fields excluding the standard set of fields (`Title`, `UserName`, `Password`, `URL`, `Notes`). So, we need to explicitly add URL here because it's not included anywhere outside of that pop-up.

**Comment by dploeger** [comment]
But username and password are already in the list?

**Comment by dploeger** [comment]
Oh and I could think of a use for notes there as well. 😊

**Comment by antelle** [comment]
Username and password are visible in the list and available as shortcuts. Let's not return Notes there, it's a long field that's not supposed to be auto-typed.

**Comment by dploeger** [comment]
Okay. Thanks!
